spk/mitmproxy/src/requirements.txt

```diff
@@ -0,0 +1,38 @@
+https://github.com/mitmproxy/mitmproxy/archive/v0.18.2.tar.gz
+backports.ssl_match_hostname>=3.5.0.1, <3.6
```

Comment:
All the versions in this file should be pinned to exactly 1 version (`==`) for reproducability reasons, otherwise we will not be able to recreate the exact same package when even 1 of these requirements updates. 
Run a `pip freeze > requirements.txt` when you've got everything installed, and use that as a base instead.